now.json

```diff
@@ -13,6 +13,11 @@
     },
     {
       "handle": "filesystem"
+    },
+    {
+      "src": "/([a-z]+)/.*",
```

Comment:
this does not catch our website names with dashes. Plus, you need to escape slashes.   

test case: https://commercetools-docs-kit-git-nm-404.commercetools.now.sh/docs-smoke-test/i-do-not-exist

```suggestion
      "src": "\/(\S+?)\/.*",
```
(any non-whitespace character, ungreedily matched until the next forward slash)

https://regex101.com/r/3ntqSz/1

Reply:
Thanks 😄